docs/user-guide/python.md

```diff
@@ -44,64 +44,78 @@ or in a submission script) using:
 
-This uses the `--user` flag to ensure the packages are installed in
-the directory specified by `PYTHONUSERBASE`.
+    source /work/t01/t01/auser/myvenv/bin/activate
 
-## Setting up virtual environments
-We recommend that you use the `pipenv` and/or `virtualenv` packages to
-manage your Python environments. A summary of how to get a virtual environment set up is contained in the below, but for further information, see:
+!!! tip
+    The `myvenv` path uses a fictitious project code, `t01`, and username, `auser`. Please remember to replace those values
+    with your actual project code and username. Alternatively, you could enter `${HOME/home/work}` in place of `/work/t01/t01/auser`.
+    That command fragment expands `${HOME}` and then replaces the `home` part with `work`.
 
-   - [Pipenv and Virtual
-     Environments](https://docs.python-guide.org/dev/virtualenvs/)
+Installing packages to your local environment can now be done as follows.
 
-Sometimes, you may need several different versions of the same package installed, for example, due to dependency issues. Virtual environments allow you to manage these conflicting requirements. The first step is to run the commands contained in the above section so that you can install the `virtualenv` package which will manage your environments. To install `virtualenv` run:
-```
-pip install --user virtualenv  # The --user flag indicates this should be installed in the user's package folder
-```
-Next you must create a folder for the virtual environment's files to live in and tell `virtualenv` to set this folder up for storing virtual environment 'stuff'. This is done by running the command
-```
-mkdir /work/t01/t01/auser/<<name of your virtual environment>>  # Create the folder
-virtualenv -p /opt/cray/pe/python/3.8.5.0/bin/python /work/t01/t01/asuser/<<name of your virtual environment>>  # -p flag means use this python interpreter
-```
-Finally, you're ready to `activate` your environment. This is done by running
-```
-source /work/t01/t01/auser/<<name of your virtual environment>>/bin/activate
-```
-Once your environment is activated you will be able to install packages as usual using `pip install <<package name>>`. These packages will only be available within this environment. When running code that requires these packages you must activate the environment, by adding the above `source ... activate` line of code to any submission scripts.
+    (myvenv) auser@ln01:~> python -m pip install <package name>
+
+Running `pip` directly as in `pip install <package name>` will also work, but we show the `python -m` approach
+as this is consistent with the way the virtual environment was created.
+
+!!! tip
+    The ARCHER2 compute nodes cannot access the `/home` file system, which means you may need to run
+    `export XDG_CACHE_HOME=${HOME/home/work}` if you're working from within an interactive session as
+    that export command will ensure the pip cache is located off `/work`.
```

Comment:
Clarification question: if I am using a login node to install packages and then want to use the environment in a job script (the most common use case), do I need to set this environment variable as described? If so, we should move this command to part of the process rather than as a tip. Or, by "interactive session", you mean interactive shell on a compute node then this is something different (but, in this case you would not be able to pull packages as you do not have internet access, so maybe I do not understand what you are getting at here?)

Also, it would make what is going on here clearer if we do not use bash substitution syntax. i.e. use `export XDG_CACHE_HOME=/work/t01/t01/user` to match what we have used previously in the commands.

Reply:
Agreed, I've removed the note regarding XDG_CACHE_HOME as it makes no sense. It could be I suppose that the user is running an interactive session on a serial node, but that's not what's being discussed in the text.